test/parallel/test-http-unix-socket.js

```diff
@@ -59,7 +59,11 @@ server.listen(common.PIPE, common.mustCall(function() {
       server.close(common.mustCall(function(error) {
         assert.strictEqual(error, undefined);
         server.close(common.mustCall(function(error) {
-          assert.strictEqual(error && error.message, 'Not running');
+          common.expectsError({
+            code: 'ERR_SERVER_NOT_RUNNING',
+            message: 'Server is not running.',
+            type: Error
+          })(error);
```

Comment:
This can be simplified to...

```js
server.close(common.expectsError({
  code: 'ERR_SERVER_NOT_RUNNING',
  message: 'Server is not running.',
  type: Error
}));
```